validator/client/src/test/java/tech/pegasys/teku/validator/client/ValidatorStatusProviderTest.java

```diff
@@ -115,11 +128,44 @@ void shouldUpdateMetricsForValidatorStatuses() {
     assertThat(gauge.getValue(ValidatorStatus.active_ongoing.name()))
         .isEqualTo(OptionalDouble.of(0));
 
-    logger.checkValidatorStatusChanges();
+    provider.onSlot(spec.computeStartSlotAtEpoch(UInt64.ONE).plus(1));
```

Comment:
maybe you should change the name of this test. It is testing that onSlot we do the job of updating statuses, and you could add the check that the subscriber is called.